docs/products/access/release-notes/1.x.md

```diff
@@ -1,5 +1,31 @@
 # Release notes 1.x versions
 
+## 1.1.1
```

Comment:
I thought we would move away from version numbers in the release notes in favour of release dates.

Reply:
I think using the release dates could be a good approach, we would most likely need to include time or use timestamp as making multiple releases within a day can definitely happen. As of now, we are still using semantic versioning so decided to comply with that standard in the release notes as well.